topostats/validation.py

```diff
@@ -223,6 +223,11 @@ def validate_config(config: dict, schema: Schema, config_type: str) -> None:
                 "core",
                 error="Invalid value in config for 'plotting.image_set', valid values " "are 'all' or 'core'",
             ),
+            "pixel_interpolation": Or(
```

Comment:
Currently this would allow `neil` as a valid option but that would then fail when it reached Matplotlib.

It should perhaps list the options so they can be validated (and if there was a desire to limit choices  to `None`, and `gaussian` this would be the place to do it).

Reply:
I'll add the options in now. I've gone for all options because if people want them they are there, and if they change the defaults, it can be expected that they understand the changes the interpolation methods incur.